lib/model/tripDatabase.dart

```diff
@@ -826,26 +826,171 @@ List<TripItem> generateTripItems() {
+          previousStep: 12,
+          nextStep: 14,
+        ),
+        StepItem(
+          imageAsset: "assets/images/trip03/trip03_mapa.jpg",
+          caption:
+              '"Własną śmiercią można własne życie tylko potwierdzić, ale zaprzeczyć mu nie można, ponieważ nasza śmierć czyni dopiero nasze życie ostatecznie nieodmiennym"\n - Leszek Kołakowski (1927-2009 r.)',
+          title: "13. Leszek Kołakowski",
+          description:
+              'W pierwszym okresie swojej działalności (mniej więcej do 1966 roku) pisał dzieła o charakterze marksistowskim, wspierając tym samym ideologię władz PRL-u. Dopiero później zaczął podejmować tematy związane z szeroko pojętą myślą chrześcijańską i historią filozofii, jednocześnie poddając krytyce ideologię marksistowską. Z tego powodu musiał emigrować z kraju - osiadł w Oksfordzie, gdzie mieszkał aż do śmierci.\n\nNa szaro-czarnym bruku tego placyku znajdują się części połamanych zabytkowych kolumn kamiennych symbolizujących nieuchronność śmierci.',
+          previousStep: 13,
+          nextStep: 15,
+        ),
+        StepItem(
+          imageAsset: "assets/images/trip03/trip03_mapa.jpg",
+          caption:
+              '"Miłować prawdę - wszelką i we wszystkim - to dla człowieka być samym sobą i ocalić samego siebie"\n - Tadeusz Styczeń (1931-2010 r.)',
+          title: "14. Tadeusz Styczeń",
+          description:
+              'Salwatorianin, etyk i wieloletni przyjaciel Karola Wojtyły oraz jego następca na katedrze Etyki KUL-u, odznaczony Orderem Odrodzenia Polski. Podejmował zagadnienia związane z ludzkim postępowaniem, które według niego powinno zawsze pozostawać wierne prawdzie rozpoznanej przez człowieka - wyjaśniał ten mechanizm m. in. na przykładzie bohaterki antycznego dramatu - Antygony. Innym z podejmowanych przezeń tematów, było usytuowanie etyki wśród wielu różnych dziedzin wiedzy jako dyscypliny naukowej.\n\nPlacyk wybrukowany małymi otoczkami, pośród których znajduje się solidnie osadzony duży głaz. Głaz ten symbolizuje prawdę, która jest jedna, stała i pewna. Tylko życie w prawdzie sprawia, że człowiek staje się prawdziwie wolny ("Poznacie prawdę, a prawda was wyzwoli" J 8,32).',
```

Comment:
whitespace typo: "m. in."
possible typo "otoczkami", I think it should be "otoczakami"